Configurations/unix-Makefile.tmpl

```diff
@@ -1271,7 +1271,8 @@ errors:
            include/openssl/ebcdic.h
            include/openssl/opensslconf.h
            include/openssl/symhacks.h ) );
-   our %cryptoheaders = ();
+   our %cryptoheaders =
+        qw( include/openssl/hpke.h );
```

Comment:
This shouldn't be necessary, it should be collected automatically by the code just a few lines down.

Reply:
yep, looks like you're right, ta - will be gone from next commit